### ISSUE TITLE
Is docs site replaced?

### ISSUE THREAD

**Issue by jacobdalamb** [zed-industries/docs#82]
Is https://docs.zed.dev/ replaced by https://zed.dev/docs/configuring-zed?

**Comment by maxdeviant** [comment]
Yes, the docs live in the [Zed repo](https://github.com/zed-industries/zed/tree/main/docs/src) now.

We do need to archive/remove this repo.
